scripts/workflow_validation_test_support.py

```
@@ -0,0 +1,217 @@
+"""Small parsed-YAML and controlled-process helpers for workflow validation."""
+
+from __future__ import annotations
+
+import os
+import re
+from pathlib import Path
+import subprocess
+import sys
+from typing import Mapping
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)